core/classes/Events/AbstractEvent.php

```diff
@@ -28,16 +28,6 @@ public static function name(): string
      */
     abstract public static function description(): string;
```

Comment:
```suggestion
   public static function description(): string
   {
       return static::name();
   }
```

no need to define this manually, wouldn't hurt to default to the name here